### DISCUSSION TITLE
GDI Graphics and the Image control.

### DISCUSSION THREAD

**Discussion by dmrvb** [twinbasic/twinbasic#798] [Show and tell]
I have a reduced version of one window from my spectrum Analyser now running in tB.
This acquires sound from soundcards or National Instruments hardware and does a real time graphical display. I can't stream to disk yet but only due to File Dialog limitations.

So, I have real time GDI graphics running in tB.

I am using image controls as the containers for the GDI output.
I appreciate that the tB Image control is not VB compatible as it has a hwnd, but its very useful.
If its going to stay like this for a while could it have a paint event please?, otherwise a refresh wipes out my graphics.
 
![image](https://user-images.githubusercontent.com/83877219/162977497-233344f2-202b-4e07-ba90-7f9cf2339029.png)

**Comment by WaynePhillipsEA** [comment]
This is cool! Thanks for sharing.

I'll be back on the GUI stuff soon.  With the kids off at the moment, I'm just completing some of the easier challenges this week, and then back to normal next week.
